pibooth/booth.py

```diff
@@ -220,10 +208,11 @@ def find_capture_event(self, events, type_filter=None):
         """Return the first found event if found in the list.
         """
         for event in events:
-            if (event.type == pygame.KEYDOWN and event.key == pygame.K_p) or \
-                    (event.type == BUTTON_DOWN and event.pin == self.button_capture):
-                if type_filter is None or type_filter == event.type:
-                    return event
+            if (event.type == pygame.KEYDOWN and event.key == pygame.K_p) and \
+                    (type_filter is None or type_filter == event.type):
+                return event
+            elif self.button_capture.is_pressed:
```

Comment:
Why the physical button (handle by gpiozero) is no longer in the type_filter?

Reply:
Same comment than for `find_settings_event`